dev/cli

```diff
@@ -191,6 +192,10 @@ function validate_gcf_url_accessible() {
 }
 
 function enable_oidc() {
+  enable_keycloak
+}
+
+function enable_keycloak() {
```

Comment:
i would call this `add_keycloak_env_vars_to_env_args` as you don't enable here keycloak